data_steward/analytics/cdr_ops/qc_serology_ct_dataset.py

```diff
@@ -38,15 +37,17 @@
 
 def get_table(table, cols, dataset, project):
     query = JINJA_ENV.from_string("""
-            SELECT {cols} 
-            FROM `{dataset}.{table}` 
-            """)
-    df = pd.read_gbq(query, project_id=project)
+            SELECT {{cols}} 
+            FROM `{{dataset}}.{{table}}` 
+            """
+    q = query.render(cols = cols, project_id=project, dataset=dataset)
+    df =execute(client, q)
     return df
 
 
-def get_data_input(rt_dataset, ct_ser_dataset, new_ct_dataset, cur_out_project,
-                   cur_project):
+def get_data_input(ct_ser_dataset, new_ser_ct_dataset, cur_out_project, cur_project):
+    
+    rt_dataset='R2020q4r1_antibody_quest'
```

Comment:
@kouamea although the dataset name doesn’t change we do not want our prod dataset names to be publicly available. can you revert this back to a parameter.